File: news_server/modules/source/infrastructure/getters/pcbeta.ts
/**
 * 远景论坛新闻源获取器
 * 从 newsnow/server/sources/pcbeta.ts 迁移
 */
import type { SourceGetter } from '../../domain/getter/getter'
import { defineRSSSource } from './utils'

export const pcbetaWindows11Getter = defineRSSSource(
  'https://bbs.pcbeta.com/forum.php?mod=rss&fid=563&auth=0'
)

export const pcbetaWindowsGetter = defineRSSSource(
  'https://bbs.pcbeta.com/forum.php?mod=rss&fid=521&auth=0'
)

File: modules/source/infrastructure/getters/pcbeta.go
package getters

import "context"

func (r *Registry) pcbetaWindows(ctx context.Context) ([]Item, error) {
	return r.rss("https://bbs.pcbeta.com/forum.php?mod=rss&fid=521&auth=0")(ctx)
}

func (r *Registry) pcbetaWindows11(ctx context.Context) ([]Item, error) {
	return r.rss("https://bbs.pcbeta.com/forum.php?mod=rss&fid=563&auth=0")(ctx)
}
